### PR TITLE
DLS-9896 - review and update mentions of government gateway

### PR DESCRIPTION
**Pull request by TomRafferty**
ran unit tests - pass
ran ATs (due to content change) - pass

### CODE REVIEW

File: test/common/DatesSpec.scala [hunk 1]
```diff
@@ -16,12 +16,11 @@
 
 package common
 
-import java.time.LocalDate
-
 import common.Dates.formatter
 import org.scalatestplus.play.guice.GuiceOneAppPerSuite
 import play.api.i18n.{Lang, Messages, MessagesApi}
 
+import java.time.LocalDate
```
Comment (karolina-zaborska) [inline test/common/DatesSpec.scala:23]: ?

File: app/views/calculation/resident/properties/checkYourAnswers/checkYourAnswers.scala.html [hunk 2]
```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  *@
 
-@import models.resident.{IncomeAnswersModel, TaxYearModel}
 @import models.resident.properties.{ChargeableGainAnswers, YourAnswersSummaryModel}
+@import models.resident.{IncomeAnswersModel, TaxYearModel}
```
Comment (karolina-zaborska) [inline app/views/calculation/resident/properties/checkYourAnswers/checkYourAnswers.scala.html:18]: ?

File: test/common/ValidationSpec.scala [hunk 3]
```diff
@@ -16,11 +16,11 @@
 
 package common
 
-import java.time.LocalDate
-
 import common.Validation._
 import play.api.data.validation.{Invalid, Valid, ValidationError}
 
+import java.time.LocalDate
+
```
Comment (karolina-zaborska) [inline test/common/ValidationSpec.scala:23]: May I know why such a decision?

File: test/common/WithCommonFakeApplication.scala [hunk 4]
```diff
@@ -17,9 +17,9 @@
 package common
 
 import org.scalatest.{BeforeAndAfterAll, Suite}
-import play.api.{Application, Play}
 import play.api.inject.guice.{GuiceApplicationBuilder, GuiceableModule}
 import play.api.test.Helpers.running
+import play.api.{Application, Play}
```
Comment (karolina-zaborska) [inline test/common/WithCommonFakeApplication.scala:22]: ?

**Review by karolina-zaborska — APPROVED** [review]

**Comment by TomRafferty** [comment]
just to clarify Karolina's questions were answered on slack and those changes were just automated :)

**Review by karolina-zaborska — APPROVED** [review]
LGTM

**Pull request merged**